src/gam.py

```diff
@@ -1213,9 +1213,34 @@ def _checkFullDataAvailable(warnings, tryDate, fullDataRequired):
           return (-1, tryDate)
   return (1, tryDate)
 
+REPORT_CHOICE_MAP = {
+  'access': 'access_transparency',
+  'accesstransparency': 'access_transparency',
+  'calendars': 'calendar',
+  'customers': 'customer',
+  'doc': 'drive',
+  'docs': 'drive',
+  'domain': 'customer',
+  'enterprisegroups': 'groups_enterprise',
+  'google+': 'gplus',
+  'group': 'groups',
+  'groupsenterprise': 'groups_enterprise',
+  'hangoutsmeet': 'meet',
+  'logins': 'login',
+  'oauthtoken': 'token',
+  'tokens': 'token',
+  'users': 'user',
+  'useraccounts': 'user_accounts',
+  }
+
 def showReport():
   rep = buildGAPIObject('reports')
   report = sys.argv[2].lower()
+  if report.replace('_', '') in REPORT_CHOICE_MAP:
+    report = REPORT_CHOICE_MAP[report.replace('_', '')]
```

Comment:
simplify to:
```
report = sys.argv[2].lower().replace('_', '')
report  = REPORT_CHOICE_MAP.get(report, report)

Reply:
If I make this change, then a new app foo_bar in the discovery doc can't be recognized as we stripped out the _ and don't have a map to put it back

Reply:
This will work, just posted:
  report = sys.argv[2].lower()
  report = REPORT_CHOICE_MAP.get(report.replace('_', ''), report)